CMakeLists.txt

```diff
@@ -199,7 +199,9 @@ function(cprover_default_properties)
      XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY ${CBMC_XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY})
 endfunction()
 
-if(CMAKE_SYSTEM_NAME STREQUAL Linux)
+if(CMAKE_SYSTEM_NAME STREQUAL Linux AND
+    (CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64" OR
+     CMAKE_SYSTEM_PROCESSOR STREQUAL "i386"))
```

Comment:
This condition means that memory analyzer will be built on Mac with Intel.
Is this the intended behaviour?

Reply:
No, that would not fulfil `CMAKE_SYSTEM_NAME STREQUAL Linux`.